lib/Github/Api/PullRequest/Review.php

```diff
@@ -159,11 +159,22 @@ public function submit($username, $repository, $pullRequest, $id, array $params
      * @param string $repository  the repository
      * @param int    $pullRequest the pull request number
      * @param int    $id          the review id
+     * @param string  $message    a mandatory dismissal message
      *
      * @return array|string
      */
-    public function dismiss($username, $repository, $pullRequest, $id)
+    public function dismiss($username, $repository, $pullRequest, $id, $message)
```

Comment:
What happens when the $message parameter is omitted (unset parameter, not just empty):
 
![capture](https://user-images.githubusercontent.com/1332071/27519148-1ec98368-59ee-11e7-86ee-f53dc8579716.PNG)